src/pages/profile/bookmarks.tsx

```diff
@@ -0,0 +1,9 @@
+import BaseNewLayout from 'layouts/base-new'
+import ProfileBookmarksView from 'views/profile/bookmarks-view'
+
+const BookmarksPage = () => {
```

Comment:
Perhaps I should confirm with design on this routing update. Lmk if I'm missing any context!

Reply:
This routing update seems good to me with the knowledge that there will _probably_ be a `/profile` page in the future! 👍🏻